MaxwellKnight/schedula
# Feat/preferences

# Add Team Preferences Management

Implements a system for team managers to set scheduling preferences that team members can choose from.

## Features
- Managers can create preference templates with date ranges
- Define available time slots and ranges
- Team members can select their preferred time slots
- Bulk time slot creation for efficiency

## Implementation
- Added backend routes for templates, time ranges, and time slots
- New frontend components (PreferencesDrawer) for preference management
- Protected routes with authentication
- Input validation for all endpoints

## Testing
1. Create preference template as manager
2. Add time ranges
3. Create time slots
4. Verify team member can select preferences
